app/controllers/users_controller.rb

```diff
@@ -1,3 +1,9 @@
 class UsersController < ApplicationController
-  expose :users, -> { User.all }
+  expose :user
+  expose_decorated(:users, -> { User.all })
```

Comment:
I haven't use decent exposure, but say me why? I use this gem for a long time and I think this works great. Besides that gem show me by Stevo. :)